scripts/objects/10-chain_whip.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const Random    = require('../../src/random').Random;
+const Broadcast = require('../../src/broadcast').Broadcast;
+const ItemUtil  = require('../../src/item_util').ItemUtil;
+const Effects   = require('../../src/effects').Effects;
+
+const util = require('util');
+
+exports.listeners = {
+
+  wield: function (l10n) {
+    return function (location, room, player, players) {
+      const toRoom = Broadcast.toRoom(room, player, null, players);
+
+      const firstPartyMessage = '<blue>You grip the chain tight in your hand, slinging it over your shoulder.</blue>';
+      const thirdPartyMessage = '<blue>' + player.getShortDesc('en') + ' grips a long heavy chain, and slings it over their shoulder.</blue>'
+      toRoom({ firstPartyMessage, thirdPartyMessage });
+
+      const missedPrerequisites = this.checkPrerequisites(player);
+
+      missedPrerequisites.forEach(prereq => {
+        switch (prereq) {
```

Comment:
In case of adding more prereqs later and to maintain style of default prereq checking...